shared/wallets/send-form/participants/to-field.js

```diff
@@ -109,32 +125,34 @@ class ToField extends React.Component<ToFieldProps, ToFieldState> {
             onChanged={this.onDropdownChange}
             items={items}
             selected={
-              this.state.selectedAccount ? (
-                <SelectedEntry account={this.state.selectedAccount} user={this.props.user} />
+              this.state.selectedRecipient instanceof Object ? (
+                <SelectedEntry account={this.state.selectedRecipient} user={this.props.user} />
               ) : (
-                undefined
+                <DropdownText key="placeholder-select" text="Pick another account" />
```

Comment:
Add some placeholder text I missed in the #13463

Reply:
This placeholder text isn't rendering for me

Reply:
Hmm what do you see when you open the send form for send to another account?